wiki/en/Installation-for-Windows.md

```diff
@@ -34,8 +35,6 @@ You can try two versions of ASIO4ALL. ASIO4ALL v2.14 includes a workaround for a
 
 [ASIO4ALL v2.14 Download](https://github.com/jamulussoftware/assets/raw/main/ASIO4ALL/v2.14/ASIO4ALL_2_14_English.exe){: .button target="_blank" rel="noopener noreferrer"}
 
-[ASIO4ALL website](https://www.asio4all.org/){: target="_blank" rel="noopener noreferrer"}
-
```

Comment:
Why did you remove this?

Reply:
We're linking to them elsewhere, and I don't want to attract their attention any more than necessary. But I also don't consider a PR complete unless I've deleted something :-)

Reply:
And `wiki/en/misc/footertext.md` wasn't enough?

Reply:
No, I had reasons to delete that :-)

Ha - I've just realised this is a link to ASIO4all not Stienberg. Sorry. Yes, we can link to them and all their ad trackers too if we really want :-)